stoplight.js

```diff
@@ -0,0 +1,32 @@
+$(".bulb").on("click", function() {
+
+var onOrNot= $(this).attr("class");
+var otherLights = $(this).siblings().attr("class");
+var onLights = $(this).siblings();
+
+console.log(onOrNot + "#");
+console.log(otherLights + "$");
+
+if (onOrNot === "bulb") {
+  $(this).addClass("active");
+} else {
+  $(this).removeClass("active");
+}
+//
```

Comment:
very much theright idea! I think you want to check to see if it has class active though